coral/src/app/router.tsx

```diff
@@ -108,6 +111,33 @@ const routes: Array<RouteObject> = [
         path: Routes.CONNECTORS,
         element: <ConnectorsPage />,
       },
+      {
```

Comment:
We should wrap that behind a feature flag, same as topic overview for now.

Reply:
No feature flag? It would be accessible by going directly to the url otherwise. Which, I admit, is not a likely scenario :D